domains/byronloarte.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Byrontosh",
+        "email": "by_tosh20@hotmail.com"
+    },
+    "records": {
+        "CNAME": "https://byrontosh.github.io"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.